src/context.cc

```diff
@@ -331,9 +331,24 @@ bool ContextBase::onStart(std::shared_ptr<PluginBase> plugin) {
 }
 
 bool ContextBase::onConfigure(std::shared_ptr<PluginBase> plugin) {
-  if (isFailed() || !wasm_->on_configure_) {
+  if (isFailed()) {
     return true;
   }
+
+  // on_context_create is yet to be executed for all the root contexts except the first one
+  if (!in_vm_context_created_ && wasm_->on_context_create_) {
+    DeferAfterCallActions actions(this);
+    wasm_->on_context_create_(this, id_, 0);
+  }
+
+  // NB: If no on_context_create function is registered the in-VM SDK is responsible for
+  // managing any required in-VM state.
+  in_vm_context_created_ = true;
```

Comment:
maybe I could do 
```cpp
if (!in_vm_context_created_) {
    in_vm_context_created_ = true;
}
```
but it seems redundant

Reply:
No, that's fine.